safe-global/web-core
# Feat: add PWA support (next-pwa)

## What it solves

Adds a framework to easily configure service workers for offline support.

See https://github.com/shadowwalker/next-pwa

## How to test

* Load the app
* Go to Chrome devtools, Network panel
* Set the network to Offline
* Navigate to different pages within the app – they should still open
